src/Bicep.Core.IntegrationTests/ScenarioTests.cs

```diff
@@ -521,7 +521,7 @@ public void Test_Issue1454()
 
             result.Template.Should().NotHaveValue();
             result.Should().HaveDiagnostics(new[] {
-                ("BCP120", DiagnosticLevel.Error, "The property \"scope\" must be evaluable at the start of the deployment, and cannot depend on any values that have not yet been calculated. You are referencing a variable which cannot be calculated in time (\"appResGrp\" -> \"rg\"). Accessible properties of rg are \"name\", \"scope\"."),
+                ("BCP120", DiagnosticLevel.Error, "The property \"scope\" must be evaluable at the start of the deployment, and cannot depend on any values that have not yet been calculated. You are referencing a variable which cannot be calculated in time (\"appResGrp\" -> \"rg\"). Accessible properties of rg are \"name\"."),
```

Comment:
Nice! Was that a specific bug fix?

Reply:
Yeah that's actually a one-liner fix - when building accessible properties, filter out write-only properties.